bee-test/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "bee-test"
+version = "0.1.0"
+authors = ["IOTA Stiftung"]
+edition = "2018"
+description = "Test utilities and feature tests for bee"
```

```suggestion
description = "Test utilities for the bee framework"
```
The feature stuff kinda was a mistake in the previous version, let's not do that.